src/vue/components/panel-compact.vue

```diff
@@ -124,59 +137,70 @@
              v-html="data.fullDefinitions"></div>
       </div>
       <div :id="inflectionsPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflections"
-           v-if="data.inflectionComponentData.inflDataReady && data.settings" v-show="inflectionsTabVisible">
-        <inflections :data="data.inflectionComponentData" :inflections-enabled="data.inflectionsEnabled"
-                     :locale="data.settings.locale.currentValue" :wait-state="data.inflectionsWaitState"
-                     @contentwidth="setContentWidth" class="alpheios-panel-inflections">
-        </inflections>
+           v-if="$store.getters[`app/hasInflData`]" v-show="inflectionsTabVisible">
+        <inflections @contentwidth="setContentWidth" class="alpheios-panel-inflections"></inflections>
       </div>
       <div :id="inflectionsBrowserPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflectionsbrowser"
-           v-if="data.inflectionBrowserEnabled && data.settings" v-show="inflectionsBrowserTabVisible">
-        <inflection-browser :data="data.inflectionBrowserData" :infl-browser-tables-collapsed="data.inflBrowserTablesCollapsed"
+           v-if="settings.contentOptions.items" v-show="$store.state.app.tabState.inflectionsbrowser">
```

Comment:
Me either, unfortunately. When I was refactoring it I removed `data.inflectionBrowserEnabled`, but left `settings.contentOptions.items` as an equivalent of `data.settings` before refactoring (was afraid to cause some damage in some tricky use cases).

I will experiment to see if removal of it will break anything or not.

Reply:
ok, i suspect that this was for unit tests, but am not sure